kitchen.vagrant.yml

```diff
@@ -9,8 +9,8 @@ driver:
   gui: false
   ssh:
     shell: /bin/sh
-  <% unless ENV['CI'] %>
   linked_clone: true
+  <% unless ENV['CI'] %>
```

Comment:
This is a change we've made across the SaltStack Formulas org, ever since @dafyddj pointed out that `linked_clone: true` worked out better in the CI as well as locally.